dataactbroker/handlers/accountHandler.py

```diff
@@ -558,23 +560,33 @@ def resetPassword(self,system_email,session):
         except Exception as e:
             exc = ResponseException("Unknown Error",StatusCode.CLIENT_ERROR,ValueError)
             return JsonResponse.error(exc,exc.status)
+
+        email = requestDict.getValue("email")
+        LoginSession.logout(session)
+        self.sendResetPasswordEmail(user, system_email, email)
+
+        # Return success message
+        return JsonResponse.create(StatusCode.OK,{"message":"Password reset"})
+
+    def sendResetPasswordEmail(self, user, system_email, email=None, inactive_user_override=False):
+        if email is None:
+            email = user.email
+
         # User must be approved and active to reset password
         if user.user_status_id != self.interfaces.userDb.getUserStatusId("approved"):
             raise ResponseException("User must be approved before resetting password", StatusCode.CLIENT_ERROR)
-        elif not user.is_active:
+        elif not inactive_user_override and not user.is_active:
```

Comment:
Intentionally left it with 2 nots as opposed to doing not (a or b) for readability